site/content/docs/main/troubleshooting.md

```diff
@@ -46,6 +46,8 @@ kubectl edit deployment/velero -n velero
 ...
 ```
 
+**Note:** Velero plugins run as binaries, i.e. after they are executed, either successful or not, they exit. So, if you see **received EOF, stopping recv loop** messages in debug logs, that does not mean an error occurred.
```

Comment:
We should clarify what it means, the question will come up eventually.

Reply:
what part?

Reply:
> So, if you see **received EOF, stopping recv loop** messages in debug logs, that does not mean an error occurred.

_What does **received EOF, stopping recv loop** mean?_

Reply:
oh got it

should be then

> **Note:** Velero plugins run as binaries, i.e. after they are executed, either successful or not, they exit. So, if you see **received EOF, stopping recv loop** messages in debug logs, that does not mean an error occurred, just that a plugin finished executing.

clear now?

Reply:
Clearer. Thanks.